jquery.flot.js

```diff
@@ -1490,7 +1490,7 @@ Licensed under the MIT license.
         function defaultTickFormatter(value, axis, precision) {
             var oldTickDecimals = axis.tickDecimals;
 
-            if (precision) {
+            if (precision && precision > 0) {
```

Comment:
Yes. This check was made initially inside the precision computation function. Since for SI notation and logaxis we adjust the precision, negative precision was needed and re-handled there, but I forgot to check the default formatter of axis...